content/doc/tutorials/build-a-java-app-with-maven.adoc

```diff
@@ -59,104 +56,96 @@ To begin this process, do either of the following, where `<your-username>` is th
 git clone https://github.com/YOUR-GITHUB-ACCOUNT-NAME/simple-java-maven-app
 ----
 
+=== Start your Jenkins instance
+
+Obtain the latest Jenkins instance, customized for this tutorial, by cloning the link:https://github.com/jenkins-docs/quickstart-tutorials.git[quickstart-tutorials repository].
+After cloning, navigate to the `quickstart-tutorials` directory, and execute the command `docker compose up -d maven` to run the example.
+Once the containers are running successfully (you can verify this with `docker compose ps`), the controller can be accessed at http://localhost:8080.
```

Comment:
It help me when running a tutorial if the steps are numbered so that I know when I've completed them.
```suggestion
1. Obtain the latest Jenkins instance, customized for this tutorial, by cloning the link:https://github.com/jenkins-docs/quickstart-tutorials.git[quickstart-tutorials repository].
2. After cloning, navigate to the `quickstart-tutorials` directory, and execute the command `docker compose up -d maven` to run the example.
3. Once the containers are running successfully (you can verify this with `docker compose ps`), the controller can be accessed at http://localhost:8080.
```

Reply:
You're right. 👍